dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/CacheHashMap.java

```diff
@@ -205,8 +250,12 @@ private void cacheInit() {
             if (trace && tc.isDebugEnabled())
                 tcInvoke(cacheStoreService.tcCacheManager, "getCache", attrCacheName, "String", "byte[]");
 
-            sessionAttributeCache = cacheStoreService.cacheManager.getCache(attrCacheName, String.class, byte[].class);
-
+            // Prevent the following warning when using Infinispan in client/server mode:
+            //[WARNING ] ISPN004005: Error received from the server: org.infinispan.server.hotrod.CacheNotFoundException: Cache with name '${attrCacheName}' not found amongst the configured caches
+            if (!"org.infinispan.jcache.remote.JCacheManager".equals(cacheStoreService.cacheManager.getClass().getName())) {
+                sessionAttributeCache = cacheStoreService.cacheManager.getCache(attrCacheName, String.class, byte[].class);
+            }
+            
             if (create = sessionAttributeCache == null) {
                 if (trace && tc.isDebugEnabled())
                     tcReturn(cacheStoreService.tcCacheManager, "getCache", "null");
```

Comment:
Same comments as earlier.  tcInvoke/tcReturn belong under the if check so that they aren't printed when we didn't actually invoke the operation.